### ISSUE TITLE
New logo/icon

### ISSUE THREAD

**Issue by mansya** [deltachat/deltachat-core#207]
Hi, I am a graphic designer, I want to help others in graphic design.

After I reviewed your project, you have no logo on this project. Therefore I want to contribute to this project by creating a new logo / icon. what do you think?

**Comment by r10s** [comment]
hi  @mansya,

help on design is very welcome :)

let me try to summarize the current state:

- in general, we have Delta Chat Logos here: [Andoroid, Website](https://github.com/deltachat/deltachat-android/blob/master/MessengerProj/src/main/res/drawable-xxxhdpi/ic_launcher.png), [iOS](https://github.com/deltachat/deltachat-ios/blob/master/images/appicon.png). This is are pixel-graphics, however, i also have the vector source somewhere.  
The logo was created by me, a _non_-designer, so i am sure, it can be improved - eg. by using a more Material-look instread of the gradients on android. Same might  be true for iOS, i do not know iOS very well.

- we just got an alternative suggestion some days ago: https://gitlab.com/mray/deltachat-pages/issues/1 (would be good if we put discussions here to not make it too hard to follow them) (in general we're also thinking about alternatives to github, but for the moment, it won't help the project).  
not sure if this is the correct time for a general discussion about a new logo, because there are much more important things to do.

- finally, we have the "intro-icons" on the webpage and when the app is started the first time (bird, rocket, muscle, gift, lock, tower). some of them came from the Telegram-source, all of them were reworked by Ilya some time ago. i think at some point we should think over, if these icons really match the targets of Delta Chat ... (@hpk42 :)

**Comment by mbeko** [comment]
> we just got an alternative suggestion some days ago: https://gitlab.com/mray/deltachat-pages/issues/1

I like the idea behind the design. It still has that straight connection to Delta Chat as Δ still is a Greek delta, just upper case (@hpk42), while a triangle is versatile in design and stands out among other mostly circular logo shapes.

**Comment by testbird** [comment]
If that suggested re-desing isn't the play? symbol, are more peer-to-peer leveled symbols really used in enough places these days? https://de.wikipedia.org/wiki/Dreieck_(Symbol)

**Comment by mbeko** [comment]
Sorry, I don't understand your question.

I don't think the delta triangle (Δ) can be confused with a play symbol (▷). The former stands on the base, the latter on a corner.

My point was that circular bubbles are often used in messenger logos (Signal, Whatsapp, Conversations, Facebook Messenger), so a triangular logo would be characteristic for Delta Chat and make it easily distinguishable from the "competition". At the same time it would keep the reference to the Greek letter delta, so it wouldn't be just a random triangle.

**Comment by testbird** [comment]
It wouldn't be just a random triangle, but dropping a small delta letter for an ubiquitous pyramidal symbol, instead of emphasizing the small delta. (To emphasize the smallness of the delta to classic email services, possibly so much, for the delta δ to actually become an epsilon ε (email chat) ;-)

**Comment by happyreacer** [comment]
I like the Icon, but I think you might want to do better, it might be a possibility with a lock.

**Comment by mbeko** [comment]
I see that you would favour a lower case delta. My comment was not about lower case versus upper case, though, but rather about circular versus triangular shape.

And concerning that, I can't think of any messenger app using a triangle (maybe Telegram coming a bit close) while I could immediately name four using the circle shape.

Anyway, just wanted to clarify for @hpk42 that the triangle still means "delta" and why I personally think the design proposition is good :slightly_smiling_face:
